e2e/components/ActionMenu.test.ts

```diff
@@ -36,30 +37,29 @@ test.describe('ActionMenu', () => {
       test.describe(theme, () => {
```

Comment:
I noticed that this test is under `Inactive items` but it's visiting the wrong ID, so updated the ID!